configs/index.json

```diff
@@ -95,6 +95,20 @@
             }
         ]
     },
+    "AMPL/USD/VWAP": {
+        "interval": "10m",
+        "endpoints": [
+            {
+                "Type": "bravenewcoin",
+                "URL": "https://bravenewcoin.p.rapidapi.com/ohlcv?size=1&indexId=551cdbbe-2a97-4af8-b6bc-3254210ed021&indexType=GWA&rapidapi-key=$RAPID_KEY",
+                "param": "$.content[0][vwap]"
+            },
+            {
+                "URL": "https://api.anyblock.tools/market/AMPL_USD_via_ALL/daily-volume?roundDay=false&debug=false&access_token=$ANYBLOCK_KEY&start=$NOW&end=$EOD",
```

Comment:
The AnyBlock API accepts `start` as the timestamp which to begin calculation of VWAP, and `end` for the end of the calculation period.

Swapping $EOD and $NOW should fix the 404 error resulting from their current orientation:
```
"https://api.anyblock.tools/market/AMPL_USD_via_ALL/daily-volume?roundDay=false&debug=false&access_token=$ANYBLOCK_KEY&start=$EOD&end=$NOW",
```

Reply:
yes this was a bug, good catch